### PR TITLE
update documentation for development/deployment:

### PR DESCRIPTION
**Pull request by naumvd95**
- requirements tab for development/deployment process added
- image-updating process note added
- small notes related to command exec

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -19,14 +19,27 @@ Overview
 
 More information about KQueen Architecture and use cases is described in `RATIONALE <RATIONALE.md>`_ file.
 
+Requirements
+-----------
+
+-  Python v3 and higher.
```
Comment (tomkukral) [inline README.rst]: We require Python 3.6+

File: README.rst [hunk 2]
```diff
@@ -39,13 +52,15 @@ Development
 
 ::
 
+    # exec into kqueen-api container
```
Comment (tomkukral) [inline README.rst]: exec in

File: README.rst [hunk 3]
```diff
@@ -39,13 +52,15 @@ Development
...
     etcdctl rm --recursive /kqueen
     ./devenv.py
 
 - Run flask shell
 
 ::
 
+    # exec into kqueen-api container
```
Comment (tomkukral) [inline README.rst]: exec in

File: README.rst (hunk 2 above)
Comment (naumvd95) [reply README.rst]: done

File: README.rst (hunk 3 above)
Comment (naumvd95) [reply README.rst]: done

**Pull request merged**